app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands = command.split()
```

Comment:
You could use unpacking and utilizing variables instead of indexing as an alternative approach for better readability
```suggestion
    command_name, input_file, output_file, *_ = command.split()
```